src/main/java/com/unbappd/enterprise/dto/Review.java

```diff
@@ -0,0 +1,13 @@
+package com.unbappd.enterprise.dto;
+import lombok.Data;
+
+public @Data
+class Review {
+    private int reviewId;
+    private int placesId;
+    private String reviewText;
+    private int reviewScore;
+    private double longitude;
+    private double latitude;
```

Comment:
Store latitude and longitude as Strings.  The Floating Point Arithmetic in doubles can cause inaccuracies here.